### DISCUSSION TITLE
Allow user to edit the values (distance, angle, or diameter) with one click

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [hlorus/CAD_Sketcher#433] [Feature Requests]
When we want to edit any value, we have to click on it to access its pop up dialog.
When the dialog appears, its default focus is on the "**Visible**" checkbox in the dialog box. 
But this is not the most frequently used field. 
The most frequently changed parameter is the dimension (distance, angle, or diameter). 
In most cases, we would not select/unselect the checkboxes at all.

Therefore, the dimension field should be shifted to the very top of the dialog.
Also, when the dialog pops up, the default focus should be on the value field, so that the user can just enter the new value and close the dialog.

**Comment by hlorus** [comment]
I agree, there's a bug in blender's api which is hindering the progress, see the issue below and the link in the comments.
https://github.com/hlorus/CAD_Sketcher/issues/259
